packages/client/src/containers/Login.container/SignupForm.js

```diff
@@ -9,14 +9,25 @@ const SignupForm = () => {
     password: '',
   });
   const [formErrors, setFormErrors] = useState({});
+  const [isSubmit, setIsSubmit] = useState(false);
+  const message = '';
+
   const handleChange = (e) => {
     setFormValues({ ...formValues, [e.target.name]: e.target.value });
   };
 
   const handleSubmit = (event) => {
+    setIsSubmit(true);
```

Comment:
submit can only be allowed **after** a conditional validation being passed for best practice